daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -35,11 +35,6 @@ func init() {
 	graphdriver.Register("btrfs", Init)
 }
 
-var (
-	quotaEnabled  = false
-	userDiskQuota = false
-)
-
```

Comment:
Think the two global variables could be avoided. the `quotaEnabled` could be wrapped into `driver *`, and `userDiskQuota` is only a temporary variable.